[2023-01-26 17:20:26,539] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.generate_platzi_data scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-01-26 17:20:26,555] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.generate_platzi_data scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-01-26 17:20:26,555] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 17:20:26,555] {taskinstance.py:1377} INFO - Starting attempt 8 of 10
[2023-01-26 17:20:26,555] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 17:20:26,581] {taskinstance.py:1397} INFO - Executing <Task(GeneratesatelliteDataOperator): generate_platzi_data> on 2023-01-03 00:00:00+00:00
[2023-01-26 17:20:26,588] {standard_task_runner.py:52} INFO - Started process 1124 to run task
[2023-01-26 17:20:26,591] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'generate_platzi_data', 'scheduled__2023-01-03T00:00:00+00:00', '--job-id', '608', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpcvq8_m3l', '--error-file', '/tmp/tmpcz68fssy']
[2023-01-26 17:20:26,594] {standard_task_runner.py:80} INFO - Job 608: Subtask generate_platzi_data
[2023-01-26 17:20:26,665] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.generate_platzi_data scheduled__2023-01-03T00:00:00+00:00 [running]> on host afaa1e0ff064
[2023-01-26 17:20:26,746] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=davidmontenegro2403@gmail.com
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=generate_platzi_data
AIRFLOW_CTX_EXECUTION_DATE=2023-01-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=8
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-03T00:00:00+00:00
[2023-01-26 17:20:26,746] {logging_mixin.py:115} INFO - La fecha l√≥gica es: 20230103
[2023-01-26 17:20:26,752] {logging_mixin.py:115} INFO - archivo creado en: /tmp/platzi_data_20230103.csv
[2023-01-26 17:20:26,767] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=generate_platzi_data, execution_date=20230103T000000, start_date=20230126T172026, end_date=20230126T172026
[2023-01-26 17:20:26,803] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-26 17:20:26,856] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
